File: Material de Apoio/FlorentinaCount.py
from pyspark.sql import SparkSession

spark = SparkSession.builder.getOrCreate()

florentinaRDD = spark.sparkContext.textFile("D:\\PyCharm\\Blueshift-DatabricksSpark\\Material de Apoio\\florentina.txt")

wordFlorentinaRDD = florentinaRDD.flatMap(lambda x: x.split(" "))

filterFlorentinaRDD = wordFlorentinaRDD.filter(lambda x: 'Florentina' in x)

filterFlorentinaRDD.count()

File: Material de Apoio/FlorentinaCount.scala
import org.apache.spark.sql.SparkSession

val spark = SparkSession.builder.getOrCreate()

val florentinaRDD = spark.sparkContext.textFile("/FileStore/tables/florentina.txt")

val wordFlorentinaRDD = florentinaRDD.flatMap(x => x.split(" "))

val filterFlorentinaRDD = wordFlorentinaRDD.filter(x =>  x.contains("Florentina"))

filterFlorentinaRDD.count()